### ISSUE TITLE
cvs_downloader: urlretrieve returns 403 forbidden when calling save().

### ISSUE THREAD

**Issue by hosaka** [EscVM/OIDv4_ToolKit#9]
I've added a print(FILE_URL) just to check the url we're trying to retrieve. At first I suspected that the missing User-Agent header might be the problem, but adding it didn't make any difference
```
    opener = urllib.request.build_opener()
    opener.addheaders = [('User-Agent','Mozilla/5.0')]
    urllib.request.install_opener(opener)

    urllib.request.urlretrieve(url, filename, reporthook)
```

Here's the error output when trying to download the missing CSV.
```
[DOWNLOAD] Do you want to download the missing file? [Y/n] y
https://storage.googleapis.com/openimages/2018_04/test\test-annotations-bbox.csv
Traceback (most recent call last):
  File "main.py", line 97, in <module>
    df_val = TTV(csv_dir, name_file)
  File "C:\wa\OIDv4_ToolKit\modules\csv_downloader.py", line 18, in TTV
    error_csv(name_file, csv_dir)
  File "C:\wa\OIDv4_ToolKit\modules\csv_downloader.py", line 43, in error_csv
    save(FILE_URL, FILE_PATH)
  File "C:\wa\OIDv4_ToolKit\modules\csv_downloader.py", line 57, in save
    urllib.request.urlretrieve(url, filename, reporthook)
  File "c:\users\march\appdata\local\programs\python\python37\Lib\urllib\request.py", line 247, in urlretrieve
    with contextlib.closing(urlopen(url, data)) as fp:
  File "c:\users\march\appdata\local\programs\python\python37\Lib\urllib\request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "c:\users\march\appdata\local\programs\python\python37\Lib\urllib\request.py", line 531, in open
    response = meth(req, response)
  File "c:\users\march\appdata\local\programs\python\python37\Lib\urllib\request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "c:\users\march\appdata\local\programs\python\python37\Lib\urllib\request.py", line 569, in error
    return self._call_chain(*args)
  File "c:\users\march\appdata\local\programs\python\python37\Lib\urllib\request.py", line 503, in _call_chain
    result = func(*args)
  File "c:\users\march\appdata\local\programs\python\python37\Lib\urllib\request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 403: Forbidden
```

**Comment by EscVM** [comment]
Are you using Windows, aren't you? :face_with_thermometer:

**Comment by hosaka** [comment]
Woah that was fast, and yes...unfortunately I am. Didn't think that'd have an effect on urllib, would it?

**Comment by EscVM** [comment]
Anyway, you have three options:
- try to fix the problem and we'd be very grateful. We'd merge your commit to the master branch.
- use a linux distribution (ubuntu, debian...)
- download from [here](https://storage.googleapis.com/openimages/web/download.html) the desired .csv file (boxes row) and put in the [csv_folder](https://github.com/EscVM/OIDv4_ToolKit/tree/master/OID/csv_folder).

**Comment by EscVM** [comment]
Perfect! Let's  see what @keldrom replies, because he wrote that piece of code.

**Comment by keldrom** [comment]
@hosaka
Thanks a lot for your appreciation :+1: 
I've found and corrected the error (obviously was caused by a bad advise from @EscVM ;) )
From line 35 of `csv_downloader.py` now is:

            if folder != 'class':
                FILE_URL = str(OID_URL + folder + '/' + file)
            else:
                FILE_URL = str(OID_URL + file)

The problem lies on the usage of `os.path.join()` function. In fact Linux uses `/` as deliminator for folder and can be used also for the urls, instead Windows use `\`. 
Btw the problem for now seems to be solved.
Thanks for your support :)
